prov/rxm/src/rxm_ep.c

```diff
@@ -770,7 +782,7 @@ static int rxm_ep_bind_cq(struct rxm_ep *rxm_ep, struct util_cq *util_cq, uint64
 {
 	int ret;
 
-	if (flags & ~(FI_TRANSMIT | FI_RECV)) {
+	if (!(flags & FI_TRANSMIT) && !(flags & FI_RECV)) {
```

Comment:
This allows unsupported flags too. Additional supported flags can be added instead.